flake.nix

```diff
@@ -99,6 +101,31 @@
             x11-gnome = runVM ./nix/vm/x11-gnome.nix;
             x11-plasma6 = runVM ./nix/vm/x11-plasma6.nix;
             x11-xfce = runVM ./nix/vm/x11-xfce.nix;
+            make-cache = let
```

Comment:
I think we should move this into a file in `nix/` and import it rather than dumping it all in our flake.